docs/INSTALLATION.md

```diff
@@ -0,0 +1,233 @@
+* `MYSQL_DATABASE` - The name of the [MariaDB](https://mariadb.org/) database which defaults to `AssetDatabase`.
+* `MYSQL_USER` - The username used to connect to the MariaDB database which defaults to `assetManager`.
+* `MYSQL_PASSWORD` - The username used to connect to the [MariaDB](https://mariadb.org/) database which defaults to `assetManager`.
+* `s3Client__AccessKey` - The access key used to connect to the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `s3Client__Secret` - The secret used to connect to the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `s3Client__ServiceURL` - The service URL of the [S3](https://www.nuget.org/packages/Amazon.S3/) compliant object storage as described in the [Asset Manager documentation](https://github.com/ove/ove-asset-services/blob/master/packages/ove-asset-manager/README.md#configuration).
+* `ServiceHostUrl` - Hostname (or IP address + port) of the corresponding Asset Management Service.
+* `MariaDB__ConnectionString` - The connection string used by the [Asset Manager](https://github.com/ove/ove-asset-services/tree/master/packages/ove-asset-manager) to connect to [MariaDB](https://mariadb.org/) database. The format of the connection string must always be similar to what is provided, but the port number, database name, username and password must change accordingly if their default values were changed.
+* `AssetManagerHostUrl` - Hostname (or IP address + port) of the [Asset Manager](https://github.com/ove/ove-asset-services/tree/master/packages/ove-asset-manager).
+
+### Running the Docker Compose builds
+
+OVE provides separate installers to help users install the necessary components. To install the core components of OVE run:
+
+```sh
+SERVICE_VERSION="latest" docker-compose -f docker-compose.ove.yml create
+```
+
+To install the [**OVE Asset Services**](https://github.com/ove/ove-asset-services) run:
+
+```sh
+SERVICE_VERSION="latest" docker-compose -f docker-compose.assets.yml create
+```
+
+Once the installation procedure has completed, the successful installation of OVE can be verified by accessing the [OVE Core API Documentation](http://OVE_CORE_HOST:PORT/api-docs) on a web browser.
+
+### Starting and stopping OVE
+
+To start the Docker runtime run:
+
+```sh
+SERVICE_VERSION="latest" docker-compose -f docker-compose.assets.yml up -d
+```
+
+To stop the Docker runtime run:
+
+```sh
+SERVICE_VERSION="latest" docker-compose -f docker-compose.assets.yml down
+```
+
```

Comment:
```suggestion
To check the docker logs of a service you can run: docker logs <service-name> (pressing **tab** before the service name should list the available services.
```

Reply:
I think this requires you have something configured to do autocompletion (otherwise the tab just lists files in the roking directory).

Running `docker ps` is a more general way to list services.

Reply:
Tab expansion does not work on MacOS. I have already resolved by including few lines on running docker ps and explaining how to obtain logs. Also added how to obtain logs on PM2.